src/components/data_comps/docn/cime_config/config_component.xml

```diff
@@ -67,13 +67,13 @@
     appropriate and is provided for testing and development purposes only.
     Users must create scientifically appropriate data for their particular
     application.  A tool is available to derive valid SOM forcing.
-    --- A setting of sst_aquapN (where N is an integer index value) is a 
-    type of prescribed SST mode used specifically for an aquaplanet setup in 
-    which global SSTs correspond to an analytic form set by the index value.  
+    --- A setting of sst_aquapN (where “N” is an integer index value) is a
```

Comment:
Please remove the "" here and below- this did not translate well in the $CASEROOT file. The quotes were removed on master - why did they get reintroduced in your PR. Did you rebase?